### PR TITLE
Feat/same homepage but better

### PR DESCRIPTION
**Pull request by taniasasaran**
This is the homepage of the application. It consists of a scrollable list of public rooms, a button for new rooms and a pair of one input and one button for joining an existent room via a roomcode.
Screenshot 2023-05-10 at 14 54 58

### CODE REVIEW

**Comment by taniasasaran** [comment]
This is the homepage of the application. It consists of a scrollable list of public rooms, a button for new rooms and a pair of one input and one button for joining an existent room via a roomcode. 
Screenshot 2023-05-10 at 14 54 58

File: assets/js/app.js [hunk 1]
```diff
@@ -22,9 +22,99 @@ import {Socket} from "phoenix"
 import {LiveSocket} from "phoenix_live_view"
 import topbar from "../vendor/topbar"
 import { Picker } from 'emoji-picker-element'
-import { Hooks } from 'hooks'
+import { Hooks } from './hooks'
 
 import "./room-modal.js"
+// let Hooks = {}
+
```
Comment (salajandan) [inline assets/js/app.js]: why is this commented?

File: assets/js/app.js [hunk 2]
```diff
@@ -22,9 +22,99 @@ import {Socket} from "phoenix"
...
+function extractActionAndMessageIdFromDomElementId(element) {
+    return element.id.split(/-(.*)/s)
+}
+
+function triggerMessageDeletionEvent(event, component) {
+    if (event.relatedTarget != undefined) {
+        elementType = event.relatedTarget.localName
```
Comment (salajandan) [inline assets/js/app.js]: what if `event.relatedTarget` is `null` ? Can it be?

File: assets/js/app.js [hunk 3]
```diff
@@ -22,9 +22,99 @@ import {Socket} from "phoenix"
...
+        messageId = extractActionAndMessageIdFromDomElementId(event.srcElement)[1]
+        document.getElementById("settings-" + messageId).setAttribute("hidden", "hidden")
+        document.getElementById("button-" + messageId).blur()
+    })
+}
+
+function adaptUtcTimestampToUserTimezone(component) {
+    dateElement = document.getElementById(component.el.id)
+    utcTimestamp = dateElement.innerHTML
+    dateElement.innerHTML = new Date(utcTimestamp).toLocaleString().replace(",", "")
```
Comment (salajandan) [inline assets/js/app.js]: Let's use `innerText` instead of `innerHTML`.

File: lib/chat_app_web/live/home_live.ex [hunk 4]
```diff
@@ -2,43 +2,43 @@ defmodule ChatAppWeb.HomeLive do
   use ChatAppWeb, :live_view
 
   def mount(_params, _session, socket) do
-    rooms = [
-      %{id: 1, name: "room1", description: "capybara numero uno"},
-      %{id: 2, name: "room2", description: "capybara numero dos"},
-      %{id: 3, name: "room3", description: "capybara numero tres"}
-    ]
+    rooms = ChatApp.Contexts.Rooms.list_rooms()
+    |> Enum.filter(fn room -> room.is_private == false end)
```
Comment (salajandan) [inline lib/chat_app_web/live/home_live.ex]: the condition can be rewritten as `not room.is_private`.
Also, the formatting seems wrong, please `mix format`.

**Review by salajandan — APPROVED** [review]
after comments

**Review by salajandan — APPROVED** [review]

**Pull request merged**